test.js

```diff
@@ -33,18 +33,22 @@ for (let i = 0; i < guides.length; i++) {
   });
 }
 
-const priceBoxes = document.querySelectorAll(".price-container");
+// target the menu cards not the price container since price container is being positioned
+const menuCards = document.querySelectorAll(".menu-card");
 
-priceBoxes.forEach((priceBox) => {
-  priceBox.addEventListener("click", () => {
-    priceBox.classList.toggle("clicked");
+// const priceBoxes = document.querySelectorAll(".price-container");
+
+menuCards.forEach((menuCard) => {
+  menuCard.addEventListener("click", () => {
+    menuCard.classList.toggle("clicked");
   });
 });
 
-let heart = document.getElementById("heart");
+// let heart = document.getElementById("heart");
```

Comment:
Commented this out because it was causing an error. might want to use query selector all here and no need for a loop since its just one item. So remove the forEach loop